addons/hr_recruitment/report/hr_recruitment_report.py

```diff
@@ -29,6 +29,8 @@ class hr_recruitment_report(models.Model):
     partner_id = fields.Many2one('res.partner', 'Partner', readonly=True)
     delay_close = fields.Float('Avg. Delay to Close', digits=(16, 2), readonly=True, group_operator="avg", help="Number of Days to close the project issue")
     last_stage_id = fields.Many2one('hr.recruitment.stage', 'Last Stage')
+    medium_id = fields.Many2one('utm.medium', 'Medium', readonly=True, help="This is the method of delivery. Ex: Postcard, Email, or Banner Ad")
```

Comment:
Method of "delivery" ? I don't understand the meaning of "delivery" in the context of job recruitment.